src/main/java/bowling/view/OutputVIew.java

```diff
@@ -0,0 +1,69 @@
+package bowling.view;
+
+import bowling.domain.frame.Frames;
+import bowling.domain.player.Player;
+
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.IntFunction;
+import java.util.stream.IntStream;
+
+public class OutputVIew {
+
+    private static final String FIRST_COLUMN = "| %s |";
```

Comment:
```suggestion
    private static final String FIRST_COLUMN = "| %7s |";
```

사실 View에 대한 부분을 UI 영역이라고 생각해서 피드백을 잘 안드리긴 하는데
현재 출력에 대해서 좀 보기가 힘든것 같습니다.
공백을 추가해 보면 좀 더 가독성이 올라갈 것 같네요. 😄 
OTHER_COLUMN 쪽도 동일 합니다.

참고) https://blog.jiniworld.me/68